css-view-transitions-2/Overview.bs

```diff
@@ -1080,8 +1081,24 @@ When capturing the old or new state for an element, perform the following steps
 
 ## Capturing and applying 'view-transition-group' ## {#vt-group-algorithm}
 
-<div algorithm="additional capture steps (group)">
-When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement| and an [=element=] |element|:
+<div algorithm="additional old apture steps (group)">
+When capturing the old or new state for an element, perform the following steps given a [=captured element=] |capturedElement|, an [=element=] |element| and a {{ViewTransition}} |transition|:
+
+	1. Let |containingGroupName| be the [=used value=] of |element|'s ''view-transition-group''.
+
+	1. If |containingGroupName| is not null, then:
+
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][containingGroupName].
```

Comment:
Don't we need a check to ensure that containingGroupName is an ancestor of this element?

Reply:
Done